src/pages/trade/ui/route-book.tsx

```diff
@@ -1,60 +1,212 @@
+import React, { useState } from 'react';
 import { useBook } from '../api/book';
 import { observer } from 'mobx-react-lite';
-import { RouteBookResponse } from '@/shared/api/server/book/types';
+import { RouteBookResponse, Trace } from '@/shared/api/server/book/types';
+import { ChevronRight } from 'lucide-react';
+import { getSymbolFromValueView } from '@penumbra-zone/getters/value-view';
+import { usePathSymbols } from '@/pages/trade/model/use-path.ts';
 
-const RouteBookLoadingState = () => {
+import { Tabs } from '@penumbra-zone/ui/Tabs';
+
+const HopCount = ({ count }: { count: number }) => {
+  return (
+    <span className={count === 0 ? 'text-white' : 'text-[#F49C43]'}>
+      {count === 0 ? 'Direct' : `${count} ${count === 1 ? 'Hop' : 'Hops'}`}
+    </span>
+  );
+};
+
+const RouteDisplay = ({ tokens }: { tokens: string[] }) => {
   return (
-    <div>
-      <div className='text-gray-500'>Loading...</div>
+    <div className='flex items-center gap-1 py-2 text-xs text-white'>
+      {tokens.map((token, index) => (
+        <React.Fragment key={token}>
+          {index > 0 && <ChevronRight className='w-3 h-3 text-gray-400' />}
+          <span>{token}</span>
+        </React.Fragment>
+      ))}
```

Comment:
issue:

1. It should display on hover. At the moment it is on-click.
2. When it is displaying, it seems to increase in height ever so slightly.